CharacterGenerator/Character.h

```diff
@@ -0,0 +1,31 @@
+//
+//  Character.h
+//  CharacterGenerator
+//
+//  Created by Henna on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface Character : NSObject
+
+//user generated
+@property (nonatomic) NSString * name;
+@property (nonatomic) int pushupLimit;
+@property (nonatomic) BOOL canFight;
+@property (nonatomic) int beerLimit;
+@property (nonatomic) UIImage * image;
+
+//we can determine this
```

Comment:
Comments should explain _why_ you're doing something. This comment actually leaves me _more_ confused, as it doesn't add to understanding.